compiler/one-cmds/one-init

```diff
@@ -143,8 +143,28 @@ def _get_parser(backends_list):
 
 
 def _verify_arg(parser, args):
-    # NYI
-    pass
+    # check if required arguments is given
+    missing = []
+    if not _utils._is_valid_attr(args, 'input_path'):
+        missing.append('-i/--input_path')
+    if not _utils._is_valid_attr(args, 'output_path'):
+        missing.append('-o/--output_path')
+    if not _utils._is_valid_attr(args, 'backend'):
+        missing.append('-b/--backend')
+
+    if _utils._is_valid_attr(args, 'model_type'):
+        if getattr(args, 'model_type') not in ['onnx', 'tflite']:
```

Comment:
if not only onnx, tflite is supported, please add TODO for supported model types.